src/components/text-input.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledInput = styled.input`
```

Comment:
We might need to switch to `textarea` in the future for multi-line comments. Another thing to think about is autoparsing links when displaying comments (e.g. using [`linkifyjs`](https://github.com/Hypercontext/linkifyjs) via [`linkify-react`](https://github.com/Hypercontext/linkifyjs/tree/main/packages/linkify-react). That can be done once we’ve created a full data cycle for single-line comments first 🙂